translations/Marathi/research/notes.md

```diff
@@ -0,0 +1,34 @@
+# Zulip Administration
+
+# Translation - Marathi
+
+## Overview:-
+
+* Marathi is a commonly used language in the state of Maharashtra in India.
+
+* This language is one of the most diverse languages in India.
+
+* Various Websites & Social Networks have added this language to their interface.
+  (e.g. Google, Facebook, Twitter to name a few)
+
+* This language is more informal, but is used formally too.
```

Comment:
Can you clairfy here what usage would be better for Zulip - formal or informal language? (What style do other internet platforms use?)